mjx/mujoco/mjx/_src/smooth.py

```diff
@@ -228,8 +228,10 @@ def factor_m(m: Model, d: Data) -> Data:
     return d
 
   # build up indices for where we will do backwards updates over qLD
-  # TODO(erikfrey): do fewer updates by combining non-overlapping ranges
   dof_madr = jp.array(m.dof_Madr)
```

Comment:
might as well kill this temporary now and switch the other references to m.dof_Madr